open-craft/opencraft
# BB-2177: Add colors and logo editor pages

This PR adds two pages to the new Ocim frontend: 
1. Colors and Preview page
2. Logos page

**Testing instructions:**
1. Checkout this branch.
2. Set `DEBUG=True` in Django to enable media urls.
2. Run Ocim (inside vagrant with `make run.dev`).
3. Go to `frontend/` and build the new API client with `./scripts/build-api-client.sh`
4. Run the new frontend: `npm start`.
5. Create a new test instance and log in the console.
6. Go to the theme preview and colors page and make sure it's working as expected (http://localhost:3000/console/theming/preview-and-colors). Compare it to the mockups.
7. Change colors and check if the values are correctly saved in the backend.
8. Go to the logos pages (http://localhost:3000/console/theming/logos). Compare it to the mockups.
9. Try uploading multiple images, check if django validation is working properly. See if the images are correctly saved in the backend.

**Reviewers:**
- [ ] @alanoe
